// create-index-html/src/usage.js
const commandLineUsage = require('command-line-usage')
let usage = ''

const sections = [
  {
    header: 'create-index-html',
    content: 'Create an index.html file with the gist of what you need to get started'
  },
  {
    header: 'Examples',
    content: [
      {
        desc: 'Create a starter index.html file',
        example: '$ create-index-html'
      },
      {
        desc: 'Use your own filename instead of "index.html"',
        example: '$ create-index-html <filename>'
      },
      {
        desc: 'You can give it the title of your page',
        example: `$ create-index-html --title='My Title'`
      }
    ]
  },
  {
    header: 'Options',
    optionList: [
      {
        name: 'help',
        description: 'Display this usage guide',
        alias: 'h',
        type: Boolean
      },
      {
        name: 'title',
        description: 'Add a title to the webpage',
        alias: 't',
        type: String
      }
    ]
  }
]

usage = commandLineUsage(sections)

module.exports = usage
